src/Areas/AzureBestPractices/AzureBestPracticesSetup.cs

```diff
@@ -19,30 +19,13 @@ public void RegisterCommands(CommandGroup rootGroup, ILoggerFactory loggerFactor
         // Register Azure Best Practices command at the root level
         var bestPractices = new CommandGroup(
             "bestpractices",
-            "Returns secure, production-grade Azure SDK best practices. Call this before generating Azure SDK code."
+            "Returns secure, production-grade Azure best practices. Call this before generating Azure SDK code."
```

Comment:
Should we update this to say something like the following?

```suggestion
            "Returns secure, production-grade Azure best practices. Call this before generating Azure SDK or Azure Functions code."
```

Reply:
The current description works. I anticipate this toolset will include other azure resources in the near future. I prefer leaving it as it is now.